cpu/stm32/periph/dma.c

```diff
@@ -445,6 +447,13 @@ int dma_configure(dma_t dma, int chan, const volatile void *src, volatile void *
     return 0;
 }
 
+void dma_transfer_complete(dma_t dma, void (*callback)(void*, dma_t), void *arg)
```

Comment:
I'd go with something less ambigious

```suggestion
void dma_set_transfer_complete_cb(dma_t dma, void (*callback)(void*, dma_t), void *arg)
```